rubin_scheduler/scheduler/detailers/detailer.py

```diff
@@ -104,6 +107,76 @@ def __call__(self, observation_list, conditions):
         return ObservationArray()
 
 
+class NSnapsFromExptimeDetailer(BaseDetailer):
+    """If exposure time is low, force number of snaps to be 1.
+
+    Parameters
+    ----------
+    min_exptime : `float`
+        The minimum exposure time. Set anything below
+        to have nexp=1. Default 29 (seconds).
```

Comment:
What is the reason to have this be 29s instead of 30? 
Is it to preserve the option to have 2 snaps in standard observing?

Reply:
I think that was an arbitrary choice. 30s is fine.